pkg/cloudprovider/providers/azure/azure_loadbalancer.go

```diff
@@ -1280,7 +1304,8 @@ func equalLoadBalancingRulePropertiesFormat(s, t *network.LoadBalancingRulePrope
 		reflect.DeepEqual(s.LoadDistribution, t.LoadDistribution) &&
 		reflect.DeepEqual(s.FrontendPort, t.FrontendPort) &&
 		reflect.DeepEqual(s.BackendPort, t.BackendPort) &&
-		reflect.DeepEqual(s.EnableFloatingIP, t.EnableFloatingIP)
+		reflect.DeepEqual(s.EnableFloatingIP, t.EnableFloatingIP) &&
+		s.IdleTimeoutInMinutes == t.IdleTimeoutInMinutes
```

Comment:
Of course this doesn't work as expected because it's a pointer.
Neither do these `DeepEqual` calls, I think.